### DISCUSSION TITLE
Its possible to Send SIP 183 when inbound call start ringing ... ?

### DISCUSSION THREAD

**Discussion by assegaf** [juha-h/baresip-studio#425] [Q&A]
Its possible to Send SIP 183 when inbound call start ringing ... ?

I wanna ask this because need something like progress info to the caller before phone ringing ...

**Comment by juha-h** [comment]
In theory yes. See https://github.com/baresip/baresip/wiki/Accounts#answermode.  Also, there would need to be a way to choose, which audio file to use with 183.
